core/ingest/src/main/java/mil/nga/giat/geowave/core/ingest/operations/KafkaToGeowaveCommand.java

```diff
@@ -204,7 +162,59 @@ public void setInputStoreOptions(
 	}
 
 	public void setInputIndexOptions(
-			List<IndexPluginOptions> inputIndexOptions ) {
+			final List<IndexPluginOptions> inputIndexOptions ) {
 		this.inputIndexOptions = inputIndexOptions;
 	}
+
+	@Override
+	public Void computeResults(
+			final OperationParams params )
+			throws Exception {
+		final String inputStoreName = parameters.get(0);
+		final String indexList = parameters.get(1);
+
+		// Config file
+		final File configFile = (File) params.getContext().get(
```

Comment:
again it looks like a slight change from previous in not using getGeoWaveConfig() ... my suspicion is that this is simply a mistake in one of the merges from master not reflecting that change rather than something that was consciously done